script.js

```diff
@@ -1,4 +1,6 @@
 const re = RegExp('^https?://');
+var sortOrder = "descending"
+var data;
```

Comment:
I made data (the JSON its loading) a global variable so that I can refresh the library.
Hopefully that is okay